src/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -126,7 +130,11 @@ INSTALLATIONS:
           - NAME: compressor3
             ENERGY_USAGE_MODEL: compressor_sampled_1d
         COMPONENT_CONDITIONS:
-          CROSSOVER: [ 0, 1, 1 ]
+          CROSSOVER:
+            - FROM: compressor2
+              TO: compressor1
+            - FROM: compressor3
+              TO: compressor1
```

Comment:
How can we support several crossovers from the same consumer? I guess we could use max_rate of the receiving consumer to decide, use crossover if receiving consumer can handle it, if not skip to next crossover or from-consumer becomes invalid.

Reply:
I guess you could make TO: a list, and then pass whatever the first consumer has room for (using get_max_rate) to the first consumer on the list, and then move to the next one with the remaining volume. I don't know how many actual use cases we would have for that, but it should at least be possible to implement like that. Or?